src/common/webview.ts

```diff
@@ -88,4 +88,17 @@ export class WebviewBase {
 	public dispose() {
 		this._disposables.forEach(d => d.dispose());
 	}
+}
+
+export class WebviewViewBase extends WebviewBase {
+	public readonly viewType: string;
+	protected _view?: vscode.WebviewView;
+
+	public show() {
+		if (this._view) {
+			this._view.show();
+		} else {
```

Comment:
When is there not a `this._view`?

Reply:
Before the webview has been resolved, so if it has been constructed but not revealed yet